src/components/form/Form.js

```diff
@@ -0,0 +1,215 @@
+      <div className="modal__content" style={{ display: displayForm ? 'block' : 'none' }}>
+        <button
+          className="modal__close-button modal__close-button_form"
+          type="button"
+          onClick={closeModal}
+        />
+        <h2 className="modal__title">Delegate us your task! Fill in this form:</h2>
+        <form className="form" onSubmit={handleSubmit} noValidate>
+          <label className="form__label form__label_type_company" htmlFor="company-name">
+            Your company name *
+            <input
+              type="text"
+              id="company"
+              name="company"
+              className="form__input form__input_type_company"
+              autoComplete="off"
+              placeholder="Company Name"
+              onChange={handleChange}
+              value={inputValues.company}
+            />
+            <span className={`form__error ${inputErrors.company && 'form__error_visible'}`}>
+              {inputErrors.company}
+            </span>
+          </label>
+
+          <label className="form__label form__label_type_email" htmlFor="email">
+            Your email *
+            <input
+              type="email"
+              id="email"
+              name="email"
+              className="form__input form__input_type_email"
+              autoComplete="off"
+              placeholder="email@email.com"
+              onChange={handleChange}
+              value={inputValues.email}
+            />
+            <span className={`form__error ${inputErrors.email && 'form__error_visible'}`}>
+              {inputErrors.email}
+            </span>
+          </label>
+
+          <label className="form__label form__label_type_description" htmlFor="description">
+            Describe your task *
+            <textarea
+              className="form__input form__input_type_description"
+              id="description"
+              name="description"
+              autoComplete="off"
+              placeholder="We want to.."
+              onChange={handleChange}
+              value={inputValues.description}
+            />
+            <span className={`form__error ${inputErrors.description && 'form__error_visible'}`}>
+              {inputErrors.description}
+            </span>
+          </label>
+
+          <label className="form__label form__label_type_email" htmlFor="name">
+            How can we call you? *
+            <input
+              type="text"
+              id="name"
+              name="name"
+              className="form__input form__input_type_company"
+              autoComplete="off"
+              placeholder="My name is"
+              onChange={handleChange}
+              value={inputValues.name}
+            />
+            <span className={`form__error ${inputErrors.name && 'form__error_visible'}`}>
+              {inputErrors.name}
+            </span>
+          </label>
+
+          <button className="form__button form__button_submit" type="submit">
```

Comment:
you make your button expand a little bit on hover. i suppose i should make my menu button do the same, huh?

Reply:
I don't think that's necessary since the menu button already has another hover effect (turning button from white to orange). I added the expand effect because I also had to apply this to the CTA-button, which has the same style as the submit button.